eclair-core/src/main/scala/fr/acinq/eclair/io/Peer.scala

```diff
@@ -54,7 +55,7 @@ class Peer(nodeParams: NodeParams, remoteNodeId: PublicKey, authenticator: Actor
         channel ! INPUT_RESTORED(state)
         FinalChannelId(state.channelId) -> channel
       }.toMap
-      goto(DISCONNECTED) using DisconnectedData(previousKnownAddress, channels)
+      goto(DISCONNECTED) using DisconnectedData(previousKnownAddress, channels, MAX_RECONNECT_INTERVAL) // when we restart, we will attempt to reconnect right away, but then we'll wait
```

Comment:
Randomizing `nextReconnectionDelay` here would help reduce the effect

Reply:
@araspitzu can you add some randomization here? Maybe, say, between 50% and 100% of `MAX_RECONNECT_INTERVAL`?

Reply:
cACK, in commit 22cb049 I added a randomized delay for the first reconnection attempt between 50% and 100% of the maximum delay ( so the first reconnection attempt will be between 30min and 1h)